sycl/doc/extensions/proposed/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -609,11 +598,6 @@ struct is_property_key_of<
 |
 Indicates the alignment of the pointer in bytes.
```

Comment:
What if the specified alignment is less than the size of the type being pointed to? Do we want to issue an error in that case?

Reply:
Yes, that makes sense to me.

Reply:
Reverted this note. See Greg's comment on this below. We want to not say this to account for cases with aggregates.